rollup.config.js

```diff
@@ -44,16 +52,12 @@ const baseConfig = {
 const baseExternal = ['react', 'prop-types', 'preact']
 
 const mainConfig = Object.assign({}, baseConfig, {
-  external: uniq(
-    baseExternal.concat(
-      flatMap(dir => {
-        const {
-          dependencies = {},
-          peerDependencies = {}
-        } = require(`${dir}/package.json`)
-
-        return [...Object.keys(dependencies), ...Object.keys(peerDependencies)]
-      }, getLernaPackages())
+  external: makeExternalPredicate(
+    uniq(
```

Comment:
@mitchellhamilton I think I've done something wrong here in the past, can't remember why i did it that way, but maybe u could look at it & verify my current reasoning

we do not have to read all lerna `package.json` files because this script bundles a single package at a time, so it is enough to read only its own `package.json` and build `external` array based on that

Reply:
I don’t totally remember why that was done but I think it might be necessary because I did a similar thing [here](https://github.com/emotion-js/next/commit/bede91841b3be2f343adc8f2da407ab987a0c297)

Reply:
👍 makes sense for peerDeps, but the removed code gathered also regular deps - maybe it was just a small mistake, gonna bring this back in a minute